app/TestAppKernel.php

```diff
@@ -15,6 +15,10 @@
 
 declare(strict_types=1);
 
-@trigger_error('The "TestAppKernel" class located at "app/TestAppKernel.php" is deprecated since Sylius 1.3. Use "Kernel" class located at "src/Kernel.php" instead.', \E_USER_DEPRECATED);
+trigger_deprecation(
+    'sylius/sylius',
+    '1.3',
+    'The "TestAppKernel" class located at "app/TestAppKernel.php" is deprecated since Sylius 1.3. Use "Kernel" class located at "src/Kernel.php" instead.',
```

Comment:
```suggestion
    'The "TestAppKernel" class located at "app/TestAppKernel.php" is deprecated. Use "Kernel" class located at "src/Kernel.php" instead.',
```
If left as is the message will be:

> Since sylius/sylius 1.3: The "TestAppKernel" class located at "app/TestAppKernel.php" is deprecated since Sylius 1.3. Use "Kernel" class located at "src/Kernel.php" instead.

Although it's a nitpick since we're updating the convention and all deprecation triggers, we might as well edit the messages.